contracts/src/v0.8/dev/automation/2_1/AutomationForwarder.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import {AutomationRegistryBaseInterface as IRegistry} from "./interfaces/AutomationRegistryInterface2_1.sol";
+import "../../../interfaces/TypeAndVersionInterface.sol";
+
+uint256 constant PERFORM_GAS_CUSHION = 5_000;
```

Comment:
I think @connorwstein had some suggestions here to implement this in a better way which would be resistant to hard forks changing gas cost of certain opcodes.

Reply:
happy to change the strategy here. For this PR I would consider it out of scope - this is the same pattern that we use in develop, just ported to a new contract